bodo/pandas/physical/project.h

```diff
@@ -183,4 +196,7 @@ class PhysicalProjection : public PhysicalSourceSink {
 
     duckdb::vector<duckdb::unique_ptr<duckdb::Expression>> exprs;
     std::shared_ptr<bodo::Schema> output_schema;
+
+    // Map of column bindings to column indices in physical input table
+    std::map<std::pair<duckdb::idx_t, duckdb::idx_t>, size_t> col_ref_map;
```

Comment:
Can you not use ColumnBinding as the key instead of pair?

Reply:
`map` requires comparison operators and `unordered_map` requires hash functions. Doesn't seem worth the trouble of getting it to work.